envoy/config/filter/http/jwt_auth/v2/config.proto

```diff
@@ -0,0 +1,168 @@
+
+syntax = "proto3";
+
+package envoy.config.filter.http.jwt_auth.v2;
+
+import "envoy/api/v2/route/route.proto";
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+// JSON Web Token (JWT) token format for authentication is defined
+// `here <https://tools.ietf.org/html/rfc7519>`_.
+// See `OAuth2.0 <https://tools.ietf.org/html/rfc6749>`_ and
+// `OIDC1.0 <http://openid.net/connect>`_ for the authentication flow.
+//
+// Example,::
+//
+//     issuer: https://example.com
+//     audiences:
+//     - bookstore_android.apps.googleusercontent.com
+//       bookstore_web.apps.googleusercontent.com
+//     jwks_uri: https://example.com/.well-known/jwks.json
+//
+message JSONWebToken {
+  // Identifies the principal that issued the JWT. See
+  // `here <https://tools.ietf.org/html/rfc7519#section-4.1.1>`_.
+  // Usually a URL or an email address.
+  //
+  // Example: https://securetoken.google.com
+  // Example: 1234567-compute@developer.gserviceaccount.com
+  //
+  string issuer = 1;
+
+  // The list of JWT `audiences
+  // <https://tools.ietf.org/html/rfc7519#section-4.1.3>`_. that are allowed to
+  // access. A JWT containing any of these audiences will be accepted.
+  //
+  // Example::
+  //
+  //     audiences:
+  //     - bookstore_android.apps.googleusercontent.com
+  //       bookstore_web.apps.googleusercontent.com
+  //
+  repeated string audiences = 2;
+
+  // URL of the provider's public key set to validate signature of the
+  // JWT. See `OpenID
+  // Discovery
+  // <https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata>`_
+  //
+  // Example: https://www.googleapis.com/oauth2/v1/certs
+  string jwks_uri = 3;
+
+  // If false, remove the JWT and do not forward to the backend.
+  bool forward_jwt = 4;
```

Comment:
Is this "true" by default? In proto3, "false" is a better choice for a default bool value.

Reply:
no, default is false.  default is not to forward already verified token to backend.